src/common/components/card/card.module.scss

```diff
@@ -53,6 +53,14 @@
     outline: 2px solid $color-black;
   }
 
+  & .clamp {
+    display: -webkit-box !important;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```

Comment:
It seems like these webkit specific flavours do not work in Firefox even though Firefox claims they should:

<img width="1680" alt="Screenshot 2022-04-11 at 7 45 18" src="https://user-images.githubusercontent.com/9090689/162667339-47c6a233-6a77-436f-98d0-6d4047dd7671.png">

Safari also works poorly:

<img width="1680" alt="Screenshot 2022-04-11 at 7 58 07" src="https://user-images.githubusercontent.com/9090689/162667762-ccc4b3e4-324c-4b0d-b9c6-ff5073dd93a0.png">

Perhaps it's the combination of rules that's the issue? Seems like this approach is a bit brittle in some of the major browsers. Perhaps we should consider alternatives like line-height/max-height?

Reply:
interesting, should be supported. Yes, there are other ways with line-height, i ll try